src/components/marketingMenuItem/marketingMenuItem.stories.tsx

```diff
@@ -57,8 +56,9 @@ export const DropdownWithUpsell: ComponentStory<typeof MarketingMenuItem> = (arg
     },
     [args],
   );
+  const handleChange = (option: Option) => console.log(option);
```

Comment:
maybe an alert is a better option

Reply:
I dont think so, it is rather invasive and we dont do it anywhere else